### DISCUSSION TITLE
Team health check feature

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#6311] [New features]
## Issue - Enhancement

[A user](https://app.hubspot.com/contacts/3888472/contact/7111951) 🔒 writes:

> Team health check feature (this one is considered to be used by two teams at least in a future)

See also: #2267 

Until we have acceptance criteria and estimated effort we'll keep this issue open and collect more feedback.

**Comment by gcrickman** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/11327401) 🔒  writes:

> After a detailed review and comparison of the Parabol tool with that of Team Retro (teamretro.com) we have decided to make Teamretro our choice of tool moving forward.
> 
> - The team health feature giving a Radar graph
> - A history of health Values for the team over time to help identity trends

**Comment by gcrickman** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/8749601) 🔒  writes:

> Hello! Thought for a feature - added screen or component for rating the sprint. The question would have to be customizable.
> Examples:
> If the team wants to get a pulse on how did the sprint goes overall and rate it 1 - 10.
> Or how did you feel we did on accomplishing our last retro items with a frown to smiley emoji face rating.

**Comment by gcrickman** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/11040351) 🔒  writes:

> We do condition a little bit the retro by providing relevant feedback and metrics to the team, so the team has all the information on the table on how things are going. I think this is a really valuable phase from the retro that I have to do outside parabol

**Comment by jordanh** [comment]
Iceboxing for now

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/19849851) 🔒  says:

> Having team radars is a pretty cool feature, spider web thing, get a significant picture, I like that a lot. We had this for onsite meetings, would be a perfect tool for Parabol

**Comment by mattkrick** [comment]
This _feels_ like feature chasing, it's more or less a survey disguised in a spider web.

IMO not important enough to dedicate current resources towards, but once we have ~10 devs I'd be happy to put one of them on it for the sake of feature parity

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/26149901) 🔒  writes: 

> Retrium or Team Retro are better tools. Better UX, more variety of retro, you can run team health checks too, which I would really want.

**Comment by jordanh** [comment]
A [former user](https://app.hubspot.com/contacts/3888472/contact/10814251) 🔒 writes:

> Parabol was perfect to be honest, but we do squad health checks and had to use another tool for that. So we decided to move our retros there too.

**Comment by jordanh** [comment]
Team Retro's model:

![image](https://user-images.githubusercontent.com/103000/134587770-7fb094e8-5b8a-4d31-af26-a56b5ea1155e.png)

**Comment by mattkrick** [comment]
if we built this by halloween we could use cute jackolanterns instead of weird smiley faces. :jack_o_lantern: 

that said, i still find this to be a "faster horse" answer to a deeper team issue, but maybe I'm wrong.
Maybe this is the first phase of a retro & once they identify the problem category they can dive into it with qualitative data?

**Reply by avivapinchas** [reply]
I've used a version of this as a check-in question on a leadership team, during our weekly team meetings. Instead of an icebreaker, we answered red/yellow/green - how were we feeling at the start of the meeting? 

I've also seen it attempted to be implemented on a few dev teams (there weren't good tools, and after a while folks stopped updating the spreadsheet 🙃 ).

Where a retro helps get qual feedback about what's going on and what could be better, something like this serves a different purpose, imo. What it did for us: 
- **Broad context or 'vibe check' on our team mates:** Sometimes folks where down for a while because they were overworked. Sometime they had stuff going on in their personal life that was taking their attention. This Q was a light-weight way to get a 'pulse', and then you could go deeper during the call or elsewhere if you wanted to. There's not another space for getting a general mood from the team. 
- **Understanding trends across teams at scale:** With bigger teams broken down into squads or permanent teams coming together to form a division, it gets harder to see the forrest for the trees in terms of, how are _we_ doing, as a group? I saw the team leads I worked with struggling to rise out of their day-to-day and get a sense of the overall mood of the team, and how it was changing over time. They cared and wanted to address issues, but it was hard to see if the overall morale was suffering, and of course, your own experience colors what you can see or notice happening with others. Having team members give a red/yellow/green indicator for their week allowed leads to get a better sense of morale and mood overall. 

I think this could definitely be an interesting phase in various different meetings - as an alternative icebreaker, as fodder for a retro. I've also been curious if we could get an analog for this by seeing how many reflections folks add in a retro to a column with a certain color - do their 'red' reflections grow? But I don't think there's enough consistency in how colors are used to do this. 

In my head, it could basically look like the estimate phase of Sprint Poker - the scale is emoji or color, instead of an 'issue', you have a single prompt or question.

[a comment by an automated account is not shown]

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/14303851) 🔒 mentioned they had attempted to use a retrospective to do a team radar-type health check.

I suggested they could create a template with the feelings they wanted to measure as columns, create reflection cards for the dimension to be measured (ex. 1-5 happiness scale), and then use votes to discover which rating was the highest for each (sort of as an average).

Is there a better way we could use templates or an add-on to provide a team health check feature?

**Comment by charlesmathias** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/16622801) 🔒  requested a Team Radar retro template as a future improvement.

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/60958151) 🔒 writes:

Secondly, is there any kind of reporting that can be done to visualise the outcome of certain types of meetings? We run quarterly health checks with our teams and it is useful to see the responses visualised as graphs such as pie charts. I presume I could create a custom meeting template to support the health check meetings but am interested to know about visualising the responses. Typically health checks would be followed by a meeting where we agree on actions that need to be taken in response to the feedback that comes out of the health checks.

**Comment by treyparabol** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/101739501) 🔒 writes:

I’m evaluating your product and I think it is great and gives its users a lot of flexibility. My team also likes it.

It would be even better if you could add another type of meeting: team health check. Parabol has a bunch of retrospective templates, but none of them fits immediately into this type of meeting. On the other hand, I know I can create my own templates, but health check is a special one, with a specific way of voting. For example, a thumb up, so-so, thumb down scheme, or a color-coded scale. Competitive products do a good job in this area, e.g.: [Team Health Check and Monitors | TeamRetro].
Do you have this template in your development plans? If so, what’s the ETA of this feature?
